docs/csharp/asynchronous-programming/index.md

```diff
@@ -263,6 +263,36 @@ The code completes the asynchronous breakfast tasks in about 15 minutes. The tot
 
 The final code is asynchronous. It more accurately reflects how a person might cook breakfast. Compare the final code with the first code sample in the article. The core actions are still clear by reading the code. You can read the final code the same way you read the list of instructions for making a breakfast, as shown at the beginning of the article. The language features for the `async` and `await` keywords provide the translation every person makes to follow the written instructions: Start tasks as you can and don't block while waiting for tasks to complete.
 
+## Async/await vs ContinueWith
+
+The `async` and `await` keywords provide syntactic simplification over using <xref:System.Threading.Tasks.Task.ContinueWith%2A?displayProperty=nameWithType> directly. At the language level, `async`/`await` is essentially syntactic sugar that the compiler transforms into continuation-based code using `ContinueWith`. However, this transformation provides significant readability and maintainability benefits, especially when chaining multiple asynchronous operations.
```

Comment:
This paragraph could be misleading for readers. The compiler isn't required to translate `await` expressions into `ContinueWith` method calls. They have similar semantics, but they aren't an exact translation.

In the final section, you correctly point out that the compiler optimizes the code generated for `await` whereas `ContinueWith` won't benefit from those improvements.